app/charts/index.spec.ts

```diff
@@ -11,6 +12,8 @@ import {
   getPossibleChartTypes,
 } from "./index";
 
+RDFCubeViewQueryMock;
```

Comment:
Is there a nicer way to doing this?

Reply:
Maybe instead ignore the eslint rule that remove it automatically in case of no use, at least the ignore would be at the place of the import and we would directly understand it.